tdrs-frontend/src/components/Profile/Profile.test.js

```diff
@@ -126,16 +127,20 @@ describe('Profile', () => {
       },
     })
 
-    render(
+    const url = 'https://idp.int.identitysandbox.gov/account'
+
+
+    const wrapper = mount(
       <Provider store={store}>
         <Profile />
       </Provider>
     )
 
-    fireEvent.click(screen.getByText('Manage Your Account at'))
-    expect(window.location.href).toBe(
-      'https://idp.int.identitysandbox.gov/account'
-    )
+    const link = wrapper.find('#loginDotGovSignIn').getElement().props[
+      'url'
+    ]
+
+    expect(link).toEqual(url)
```

Comment:
Now that we are not using window.location.href it is easier to test the links this way